docker-compose.prod.yml

```diff
@@ -19,6 +19,7 @@ services:
       - DOWNLOAD_FILE_TARGET_DIR=finished-files/
       - UPLOAD_FILE_TARGET_DIR=to-transcribe/
       - FINISHED_FILE_FORMAT=.zip
+      - LOG_LEVEL=INFO
```

Comment:
Add this to the `Dockerfile`, too.
We should slim these docker-compose-files up. You do not need all the environment variables.